apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsArrayControl.tsx

```diff
@@ -37,7 +38,9 @@ import DraggableDrawerButton from "./DraggableDrawerButton"
 
 export const jsonFormsArrayControlTester: RankedTester = rankWith(
   JSON_FORMS_RANKING.ArrayControl,
-  isObjectArrayControl,
+  (uischema, schema, context) =>
+    isObjectArrayControl(uischema, schema, context) ||
+    isPrimitiveArrayControl(uischema, schema, context),
```

Comment:
issue: I think this component wasn't designed for array of primitives, so might need to have a separate renderer

Reply:
it actually worked (refer to video) so i think it might be fine tbh